Project.toml

```diff
@@ -34,4 +34,4 @@ julia = "1.6"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["Test"]
+test = ["Test"]
```

Comment:
this is the sort of whitespace change it's nice to not include in a code review...